### PR TITLE
Removed references to Worldview Timeline Components. Updated roadmap …

### PR DESCRIPTION
**Pull request by mike-mcgann**
Fixes nasa-gibs/worldview#727.

Changes in this pull request: 
- Changed references Worldview Timeline Components to Worldview Components
- Updated link to the current roadmap.

@nasa-gibs/worldview

### CODE REVIEW

**Review by minniewong — COMMENTED** [review]
I think you should ask Ryan to review, since he's the one who asked for the changes :)

File: README.md [hunk 1]
```diff
@@ -87,6 +87,6 @@ See [LICENSE.md](LICENSE.md)
 
 We'd be quite excited if you'd like to contribute to Worldview Timeline Components! Whether it's finding bugs, adding new features, fixing anything broken, or improving documentation, get started by submitting an issue or pull request!
```
Comment (rboller) [inline README.md]: Need to change one more reference to "Worldview Timeline Components" here in line 88

**Review by rboller — CHANGES_REQUESTED** [review]
One more reference to "Worldview Timeline Components" needs to be updated

**Comment by mike-mcgann** [comment]
Good catch. Check now.

**Review by rboller — APPROVED** [review]
Looks good!

**Review by minniewong — APPROVED** [review]

**Pull request merged**